juspay/hyperswitch
# refactor(users): Make password nullable in `users` table

## Type of Change



## Description

This API will make the password column in the `users` table nullable.

### Additional Changes





## Motivation and Context

Closes #4901.

## How did you test it?

1. Create user with `connect_account` API
	```
	curl --location 'http://localhost:8080/user/connect_account' \
	--header 'Content-Type: application/json' \
	--data-raw '{
	    "email": "unregistered email"
	}'
	```

2. Invite a unregistered user
	```
	curl --location 'http://localhost:8080/user/user/invite_multiple' \
	--header 'Content-Type: application/json' \
	--header 'Authorization: Bearer JWT' \
	--data-raw '[
	    {
	        "email": "new user email",
	        "name": "name",
	        "role_id": "merchant_view_only"
	    }
	]'
	```

3. Check the DB entry for those users using the following query
	```sql
	select * from users where email = 'email used in previous API'
	```
	Password should be null in all the cases.

## Checklist
